src/colmap/sfm/incremental_mapper.cc

```diff
@@ -108,6 +108,11 @@ IncrementalMapper::IncrementalMapper(
       num_total_reg_images_(0),
       num_shared_reg_images_(0) {}
 
+void IncrementalMapper::SetFixedImageIds(
```

Comment:
Call this `SetExistingImageIds` and move the declaration+definition closer to `ExistingImageIds`